docs/guides/setting-url-wordpress-source-data..md

```diff
@@ -0,0 +1,114 @@
+# Setting the URL of the WordPress source of data
+
+The most important setting in a Frontity project is the WordPress installation that can be used as the source of data
+
+The format of the WordPress URL will be different depending on the type of WordPress installation used with the Frontity project so the type of WordPress installation may require different ways of setting this url in the Frontity settings
+
+The main properties in the settings of this package that are implied in the URL of the WordPress source of data are:
```

Comment:
Replace
"The main properties in the settings of this package that are implied in the URL of the WordPress source of data are:"
with
"The main properties in the settings of this package where the values are determined by the type of WordPress installation used as the source of data are:"